tests/language-feature/generics/dependent-generic-3.slang

```diff
@@ -0,0 +1,32 @@
+//TEST(compute):COMPARE_COMPUTE(filecheck-buffer=CHECK): -shaderobj -output-using-type
+//TEST(compute):COMPARE_COMPUTE(filecheck-buffer=CHECK): -vk -shaderobj -output-using-type
+
+//TEST_INPUT:ubuffer(data=[0 0 0 0], stride=4):out,name=outputBuffer
+RWStructuredBuffer<int> outputBuffer;
+
+interface IOperation<T : __BuiltinFloatingPointType> // <== This is the problem
```

Comment:
what is the problem?

Reply:
This was copied from a previous GitHub issue, I need to remove the commment. There is no problem.